src/main/java/ru/hogwarts/school/service/StudentService.java

```diff
@@ -44,4 +60,20 @@ public Student remove (long id) {
     public List<Student> getStudentByAge(int age) {
         return studentRepository.getStudentByAge(age);
     }
+
+    public List<Student> findByAgeBetween(int minAge, int maxAge) {
+        return studentRepository.findByAgeBetween(minAge, maxAge);
+    }
+
+    private Faculty findFaculty(Long facultyId) {
+        Faculty faculty = null;
+        faculty = facultyRepository.findById(facultyId)
+                .orElseThrow(() -> new FacultyNotFoundException(facultyId));
+        return faculty;
```

Comment:
```suggestion
        return facultyRepository.findById(facultyId)
                .orElseThrow(() -> new FacultyNotFoundException(facultyId));
```
зачем лишние строчки кода?